tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand.cfg

```diff
@@ -28,11 +28,15 @@
 # Copyright (c) 2012 by Delphix. All rights reserved.
 #
 
+#
+# Copyright (c) 2017 Lawrence Livermore National Security, LLC.
+#
+
 
-export org_size=1g
-export exp_size=2g
+export org_size=100m
+export exp_size=200m
```

Comment:
When #5679 get's merged (very soon!) you'll want to use $MINVDEVSIZE for this.

```
org_size="$((($MINVDEVSIZE / (1024 * 1024)) * 100))m"
```

Not that I'm opposed to it but what's the motivation for reducing the file size if they're sparse?